src/accelerate/utils/random.py

```diff
@@ -30,7 +31,8 @@
 
 def set_seed(seed: int, device_specific: bool = False):
     """
-    Helper function for reproducible behavior to set the seed in `random`, `numpy`, `torch`.
+    Helper function for reproducible behavior to set the seed in `random`, `numpy`, `torch`. Also will set the random
+    seed for `SeedableRandomSampler` through the `ACCELERATE_SEED` environment variable.
```

Comment:
Similar comment to the one above: Users don't really care about `SeedableRandomSampler`, as this is an implementation detail of accelerate. They will care more about knowing when this function needs to be used.